www/docusaurus.config.js

```diff
@@ -48,14 +48,14 @@ module.exports = {
       // contextualSearch: true,
       // searchParameters: {},
     },
-    announcementBar: {
-      id: 'v10',
-      content:
-        "🚀 You are looking at tRPC <strong>version 10</strong>! Read the <a href='/blog/announcing-trpc-10'>announcement post</a> or see the <a href='/docs/migrate-from-v9-to-v10'>migration guide</a> if you're currently using tRPC v9",
-      backgroundColor: 'var(--ifm-color-primary-dark)',
-      textColor: '#ffffff',
-      isCloseable: true,
-    },
+    //     announcementBar: {
+    //       id: 'v10',
+    //       content:
+    //         "🚀 You are looking at tRPC <strong>version 10</strong>! Read the <a href='/blog/announcing-trpc-10'>announcement post</a> or see the <a href='/docs/migrate-from-v9-to-v10'>migration guide</a> if you're currently using tRPC v9",
+    //       backgroundColor: 'var(--ifm-color-primary-dark)',
+    //       textColor: '#ffffff',
+    //       isCloseable: true,
+    //     },
```

Comment:
Maybe we just delete it entirely? We can always look back at the old code when it comes to v11, if we need the template